test/module/irohad/ordering/ordering_service_test.cpp

```diff
@@ -58,6 +82,13 @@ class OrderingServiceTest : public OrderingTest {
   std::thread loop_thread;
   ordering::MockOrderingGate *fake_gate;
   std::unique_ptr<iroha::ordering::proto::OrderingService::Stub> client;
+  std::condition_variable cv;
+  std::mutex m;
+
+ private:
+  std::atomic_bool _b;
```

Comment:
https://google.github.io/styleguide/cppguide.html#Variable_Names
> Data members of classes, both static and non-static, are named like ordinary nonmember variables, but with a trailing underscore.

Reply:
thanks! will fix that